applications/nrf5340_audio/doc/user_interface.rst

```diff
@@ -19,14 +19,16 @@ Switches
 
 The application uses the following switches on the supported development kit:
 
-+-------------------+-------------------------------------------------------------------------------------+---------------------------------------+
-| Switch            | Function                                                                            | Applications                          |
-+===================+=====================================================================================+=======================================+
-| **POWER**         | Turns the development kit on or off.                                                | All                                   |
-+-------------------+-------------------------------------------------------------------------------------+---------------------------------------+
-| **DEBUG ENABLE**  | Turns on or off power for debug features.                                           | All                                   |
-|                   | This switch is used for accurate power and current measurements.                    |                                       |
-+-------------------+-------------------------------------------------------------------------------------+---------------------------------------+
++-------------------+-------------------------------------------------------------------------------------+---------------------------------------+-------------------------------+
+| Switch            | Function                                                                            | Applications                          | Development Kit               |
+|                   |                                                                                     |                                       +------------------+------------+
+|                   |                                                                                     |                                       | nRF5340 Audio DK | nRF5340 DK |
++===================+=====================================================================================+=======================================+==================+============+
+| **POWER**         | Turns the development kit on or off.                                                | All                                   |         Y        |      Y     |
++-------------------+-------------------------------------------------------------------------------------+---------------------------------------+------------------+------------+
+| **DEBUG ENABLE**  | Turns on or off power for debug features.                                           | All                                   |         Y        |      N     |
```

Comment:
https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/nrf5340_audio/doc/requirements.html are not mentioning the nRF5340 DK as supported platform for the audio applications, so you cannot add mentions of the nRF5340 DK to the docs until that page is updated. This is because the documentation will become inconsistent and confusing to users.

Reply:
Added nRF5340 DK as a supported platform.

Reply:
Unfortunately, we need to revert to the old formatting. See my other comment.